pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1790,3 +1853,16 @@ func getSidecarInboundBindIP(node *model.Proxy) string {
 	}
 	return defaultInboundIP
 }
+
+func ModelProtocolToListenerProtocol(node *model.Proxy, protocol config.Protocol) plugin.ListenerProtocol {
```

Comment:
Is this method needed to avoid a circular dependency with the plugin package?

Reply:
Yes. I will move it to plugin package.